src/Pcbnew/Pcbnew_create_and_modify_board.adoc

```diff
@@ -34,6 +34,47 @@ The resulting outline might look something like this:
 
 image:images/Pcbnew_simple_board_outline.png[]
 
+Importing a DXF drawing for a board outline
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+As an alternative to drawing the board outline in Pcbnew directly, a 
+board outline can also be imported from a DXF drawing. This can be used to 
+create more complex board or allows board shapes to be defined in other 
```

Comment:
there is something about the board(s) and s' that is not right here. Please fix.